smac/epm/rf_with_instances.py

```diff
@@ -62,9 +58,11 @@ def __init__(self, types,
                  max_depth=20,
                  eps_purity=1e-8,
                  max_num_nodes=1000,
-                 seed=42):
+                 seed=42,
+                 **kwargs):
+
+        super().__init__(**kwargs)
```

Comment:
This will fail if instance_features and num_pca_dim are not given. Is this intended?